chapters/en/slides/module1/module1_31.Rmd

```diff
@@ -194,17 +266,36 @@ chart1
 ```{python include =FALSE}
 chart1.save(path + 'chart1.png')
 ```
+>>>>>>> 30a2af7617fb9b7197f738c4284608ee868b5be3
 
-<img src="/module1/chart1.png" alt="A caption" width="30%" />
+Also every good graph should have a title! A title provides useful information as to what the plot is about. Let's take this opportunity to finish off our scatter plot graph and set the argument `title` to something as well. To set the `title`, we use the `title` argument to the `.properties(title=..)` function.
 
 ```{python}
+# lets set the opacity to 30%
+chart6 = alt.Chart(df, width=500, height=300).mark_circle(color='red', size=12, opacity=0.3).encode(
+    x='sugars',
+    y='calories'
+).properties(title="Scatter plot sugars vs calories for different cereals")
+chart6
+```
 
+
+```{python include =FALSE}
+chart6.save(path + 'chart6.png')
 ```
 
+<img src="/module1/chart6.png" width="45%" />
 
+Notes: Script here.
+<html>
+<audio controls >
+  <source src="/placeholder_audio.mp3" />
+</audio></html>
 
 ---
 
+
+
 # Let’s apply what we learned!
 
 Notes: Script here
```

Comment:
This was pretty good Elijah! 2 things to add slides -> height and width arguments and I think we should upfront teach them about N O Q for the x and y variables.